Test_Engine/Query/HasSingleNamespace.cs

```diff
@@ -1,43 +0,0 @@
-﻿/*
```

Comment:
Because it's not in `Checks`, it's a redundant method that's not called anywhere. `Test_Engine\Query\Checks\HasSingleNamespace.cs` is still there,  is what is called by `Test_Test` and doesn't defer to this method.

Reply:
Ah right ok, makes sense! Resolved :smile: